mail/alpine/Portfile

```diff
@@ -3,23 +3,23 @@
 PortSystem          1.0
 
 name                alpine
-version             2.21
+version             2.22
 revision            1
 categories          mail
 license             Apache-2
 maintainers         netpurgatory.com:john gmail.com:jerryyhom openmaintainer
 platforms           darwin
 
-homepage            https://web.archive.org/web/20170627061225/http://alpine.freeiz.com
+homepage            http://alpine.x10host.com/
 description         alpine - a Program for Internet News and Email
 long_description    Alpine is a tool for reading, sending, and \
                     managing electronic messages that was designed \
                     with novice users in mind.
 
-master_sites        http://www.netpurgatory.com/alpine
+master_sites        http://alpine.x10host.com/alpine/release/
```

Comment:
It looks like this needs to be:

`master_sites        http://alpine.x10host.com/alpine/release/src`

Reply:
Yeah, I missed the `src`.  My fault for rushing.